src/Confluent.Kafka/ConfigPropertyNames.cs

```diff
@@ -0,0 +1,179 @@
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///     Names of all configuration properties specific to the
+    ///     .NET Client.
+    /// </summary>
+    public static class ConfigPropertyNames
+    {
+        // ---- Producer
+
+        /// <summary>
+        ///     Specifies whether or not the producer should start a background poll 
+        ///     thread to receive delivery reports and event notifications. Generally,
+        ///     this should be set to true. If set to false, you will need to call 
+        ///     the Poll function manually.
+        /// 
+        ///     default: true
+        /// </summary>
+        public const string EnableBackgroundPollPropertyName = "dotnet.producer.enable.background.poll";
+
+        /// <summary>
+        ///     Specifies whether to enable notification of delivery reports. Typically
+        ///     you should set this parameter to true. Set it to false for "fire and
+        ///     forget" semantics and a small boost in performance.
+        /// 
+        ///     default: true
+        /// </summary>
+        public const string EnableDeliveryReportsPropertyName = "dotnet.producer.enable.delivery.reports";
+
+        /// <summary>
+        ///     A comma separated list of fields that may be optionally set in delivery
+        ///     reports. Disabling delivery report fields that you do not require will
+        ///     improve maximum throughput and reduce memory usage. Allowed values:
+        ///     key, value, timestamp, headers, all, none.
+        /// 
+        ///     default: all
+        /// </summary>
+        public const string DeliveryReportEnabledFieldsPropertyName = "dotnet.producer.delivery.report.enabled.fields";
```

Comment:
the property above is "enable", while this one is "enabled". Decide on one, preferably the former since that is what is used in other configs.
So perhaps:
`dotnet.producer.enable.delivery.report.fields=..`

Reply:
i'll drop the word enabled altogether, which is better than either of our suggestions. this scenario is different to the one above, and I don't think the word enable is appropriate.